src/backend/cuda/subconfigure.m4

```diff
@@ -90,6 +92,29 @@ if test "$with_cuda" != "no" ; then
             have_cuda=no
```

Comment:
How about move the retest here?

Reply:
Did you mean it this way? I now additionally set the host compiler explicitly to gcc instead of relying on the default.